easybuild/easyconfigs/f/FHI-aims/FHI-aims-200112_2-intel-2019b.eb

```diff
@@ -0,0 +1,49 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Dugan Witherick (University of Warwick)
+#
+##
+easyblock = 'CMakeMake'
+
+name = 'FHI-aims'
+version = '200112_2'
+
+homepage = 'https://aimsclub.fhi-berlin.mpg.de/'
+description = """FHI-aims is an efficient, accurate all-electron,
+full-potential electronic structure code package for computational molecular
+and materials science (non-periodic and periodic systems). The code supports
+DFT (semilocal and hybrid) and many-body perturbation theory. FHI-aims is
+particularly efficient for molecular systems and nanostructures, while
+maintaining high numerical accuracy for all production tasks. Production
+calculations handle up to several thousand atoms and can efficiently use (ten)
+thousands of cores. 
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'opt': True, 'precise': True}
+
+# The source code must be downloaded manually from the FHI-aims club.
```

Comment:
@dithwick maybe worth mentioning that you need to sign and return the license form, for the download.

Reply:
I'll expand the text, there are voluntary fees for the academic license so I don't want to phrase it in such a way that people skip this possible option.

Reply:
@verdurin I've expanded the text, is that the sort of thing you were looking for?

Reply:
Thanks.